src/app/AdmittingView.fxml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import java.lang.*?>
+<?import java.util.*?>
+<?import javafx.scene.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+
+<AnchorPane id="AnchorPane" prefHeight="400.0" prefWidth="600.0" xmlns:fx="http://javafx.com/fxml/1" xmlns="http://javafx.com/javafx/2.2" fx:controller="app.AdmittingViewController">
+  <children>
+    <VBox id="VBox" alignment="CENTER" layoutX="116.0" layoutY="100.0" prefHeight="218.0" prefWidth="356.0" spacing="5.0">
```

Comment:
It would be nice to use columns here in the future so everything lines up nicely.